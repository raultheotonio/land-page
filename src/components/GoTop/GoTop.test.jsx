import { screen } from '@testing-library/react';
import { GoTop } from '.';
import renderTheme from '../../utils/RenderTheme';

describe('<GoTop />', () => {
  it('should render', () => {
    renderTheme(<GoTop />);

    expect(screen.getByRole('link', { name: 'Go to top' })).toBeInTheDocument();
    expect(screen.getByRole('link', { name: 'Go to top' })).toHaveAttribute('href', '#');

    expect(screen.getByRole('link', { name: 'Go to top' })).toMatchSnapshot();
  });
});
