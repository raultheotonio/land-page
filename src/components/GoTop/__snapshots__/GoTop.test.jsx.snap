// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GoTop /> should render 1`] = `
<a
  aria-label="Go to top"
  class="sc-bcXHqe iEMdUU"
  href="#"
  title="go to top"
>
  <svg
    aria-hidden="true"
    class="StyledIconBase-sc-ea9ulj-0 dXOYdB"
    fill="currentColor"
    focusable="false"
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M7.41 15.41 12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
    />
  </svg>
</a>
`;
